### PR TITLE
feat(plugin): add functionality to about data button

### PR DESCRIPTION
**Pull request by nourbalaha**
Done:
- Add functionality to `about data` button in the sidebar.

### CODE REVIEW

File: plugin/web/extensions/sidebar/core/components/content/common/DatasetCard/index.tsx [hunk 1]
```diff
@@ -73,7 +73,18 @@ const DatasetCard: React.FC<Props> = ({
           });
         },
       },
-      { id: "about", title: "About Data", icon: "about", value: "www.plateau.org/data-url" },
+      {
+        id: "about",
+        title: "About Data",
+        icon: "about",
+        value: "www.plateau.org/data-url",
```
Comment (KaWaite) [inline plugin/web/extensions/sidebar/core/components/content/common/DatasetCard/index.tsx:80]: This value isn't needed anymore.

File: plugin/web/extensions/sidebar/core/components/content/common/DatasetCard/index.tsx [hunk 2]
```diff
@@ -195,7 +206,7 @@ const DatasetCard: React.FC<Props> = ({
                   </LeftMain>
                   <ArrowIcon icon="arrowDown" size={16} expanded={expanded} />
                 </HeaderContents>
-                {inEditor && expanded && (
+                {expanded && (
```
Comment (KaWaite) [inline plugin/web/extensions/sidebar/core/components/content/common/DatasetCard/index.tsx:209]: The tabs should only be shown inEditor. Why did you remove the inEditor check here and with the add field and save buttons?

File: plugin/src/sidebar.ts [hunk 3]
```diff
@@ -253,6 +253,9 @@ reearth.on("message", ({ action, payload }: PostMessageProps) => {
   } else if (action === "modalClose") {
     reearth.modal.close();
     welcomePageIsOpen = false;
+  } else if (action === "openDatasetDetails") {
+    reearth.modal.show(dataCatalogHtml, { background: "transparent" });
+    reearth.modal.postMessage({ action: "openDatasetDetails", payload });
```
Comment (KaWaite) [inline plugin/src/sidebar.ts:258]: Sorry can you use the catalogModalOpen action, and just do a check 
```
if(payload){
   reearth.modal.postMessage({ action: "openDatasetDetails", payload });
}
```

Comment (KaWaite) [reply plugin/src/sidebar.ts:258]: We already have so many actions. Don't want to add more unless completely necessary..

File: plugin/web/extensions/sidebar/core/components/content/common/DatasetCard/index.tsx (hunk 1 above)
Comment (nourbalaha) [reply plugin/web/extensions/sidebar/core/components/content/common/DatasetCard/index.tsx:80]: Done!

File: plugin/web/extensions/sidebar/core/components/content/common/DatasetCard/index.tsx (hunk 2 above)
Comment (nourbalaha) [reply plugin/web/extensions/sidebar/core/components/content/common/DatasetCard/index.tsx:209]: This was made for developement purposes.
Done!

File: plugin/src/sidebar.ts (hunk 3 above)
Comment (nourbalaha) [reply plugin/src/sidebar.ts]: Done!

**Review by KaWaite — APPROVED** [review]
